env.cpu.mac.yml

```diff
@@ -9,7 +9,5 @@ dependencies:
     - torch-scatter
     - torch-sparse
     - torch-spline-conv
-    - openvino==2021.2
-    # Use after merge of https://github.com/openvinotoolkit/openvino/pull/3123
-    # - git+git://github.com/openvinotoolkit/openvino.git#subdirectory=model-optimizer
-    - git+git://github.com/dkurt/openvino.git@mo_pip#subdirectory=model-optimizer
+    - openvino==2021.3
```

Comment:
Seems like there is an issue with Python 3.7 on Ubuntu 20.04. Let's try to revert on 2021.2